core/modules/layout/css/layout.flexible.admin.css

```
@@ -15,22 +15,27 @@
   padding: 4px 0 4px 12px;
 }
 
+.flexible-row .layout-editor-region {
+  border: 1px solid #EAEAEA;
+}
+
 .flexible-row .layout-editor-region-title .label {
   font-size: 16px;
 }
 
 .layout--flexible .layout-editor-block {
   box-shadow: none;
+  cursor: row-resize;
 }
 .layout--flexible .layout-editor-block-title {
   padding: 0.4em 1em;
-  margin-bottom: 10px;
+  margin:  0 -.9375rem .9375rem;
```

Extra space needs to be removed here:
```suggestion
  margin: 0 -.9375rem .9375rem;
```

---

@klonos Good catch. I saw that before and thought i had fixed it. Apparently not.